### PR TITLE
fix: docstring in `Attributes.lean`

### PR DESCRIPTION
**Pull request by znssong**
Changing document string in `Attributes.lean`, in order to consistent with code in `Lean.Parser.Attr`.

### CODE REVIEW

File: src/Lean/Util/PtrSet.lean [hunk 1]
```diff
@@ -21,7 +21,7 @@ unsafe instance : BEq (Ptr α) where
 /--
 Set of pointers. It is a low-level auxiliary datastructure used for traversing DAGs.
 -/
-unsafe def PtrSet (α : Type) :=
+unsafe abbrev PtrSet (α : Type) :=
```
Comment (semorrison) [inline src/Lean/Util/PtrSet.lean:24]: This seems unrelated?

File: src/Lean/Attributes.lean [hunk 2]
```diff
@@ -67,11 +67,10 @@ def registerBuiltinAttribute (attr : AttributeImpl) : IO Unit := do
   Helper methods for decoding the parameters of builtin attributes that are defined before `Lean.Parser`.
   We have the following ones:
   ```
-  @[builtin_attr_parser] def simple     := leading_parser ident >> optional ident >> optional priorityParser
-  /- We can't use `simple` for `class`, `instance`, `export` and `macro` because they are  keywords. -/
-  @[builtin_attr_parser] def «class»    := leading_parser "class"
-  @[builtin_attr_parser] def «instance» := leading_parser "instance" >> optional priorityParser
+  @[builtin_attr_parser] def simple     := leading_parser ident >> optional (ppSpace >> (priorityParser <|> ident))
+  /- Remark, We can't use `simple` for `class`, `instance`, `export`, and `macro` because they are keywords. -/
```
Comment (semorrison) [inline src/Lean/Attributes.lean:71]:
```suggestion
```

File: src/Lean/Attributes.lean [hunk 3]
```diff
@@ -67,11 +67,10 @@ def registerBuiltinAttribute (attr : AttributeImpl) : IO Unit := do
...
   @[builtin_attr_parser] def «macro»    := leading_parser "macro " >> ident
+  @[builtin_attr_parser] def «export»   := leading_parser "export " >> ident
   ```
   Note that we need the parsers for `class`, `instance`, and `macros` because they are keywords.
```
Comment (semorrison) [inline src/Lean/Attributes.lean:75]:
```suggestion
  Note that we need the parsers for `class`, `instance`, `export` and `macros` because they are keywords.
```

File: src/Lean/Util/PtrSet.lean (hunk 1 above)
Comment (meow-sister) [reply src/Lean/Util/PtrSet.lean:24]: Oh, sorry. I did another modification that needed to write in a new PR.

Comment (meow-sister) [reply src/Lean/Util/PtrSet.lean]: Now I fixed this. I moved the `PtrSet` modification to another branch.
